### ISSUE TITLE
FSDevTools requires Java 8 but FirstSpirit runs with Java 11

### ISSUE THREAD

**Issue by m-kotowski** [e-Spirit/FSDevTools#31]
Hi.

In my local development environment, I run FirstSpirit with Java 11. I also use FSDevTools with the same Java version resulting in the following warnings. Do you plan to support Java 11 or newer for the FSDevTools in the near future?

```
 INFO fs-cli version 2.5.2 / git hash ea7eea2d178da11be820d4ca8d51add88daa184f
 INFO Build for FirstSpirit version 5.2.190710
 INFO Using FirstSpirit Access API version 5.2.190808.78484 (fs-isolated-runtime.jar)
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.espirit.moddev.cli.reflection.ReflectionUtils (file:/home/michael/fs-cli/lib/fsdevtools-cli-2.5.2.jar) to field sun.reflect.annotation.AnnotationInvocationHandler.memberValues
WARNING: Please consider reporting this to the maintainers of com.espirit.moddev.cli.reflection.ReflectionUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by kafmann-e-spirit** [comment]
Hi Michael,

we will stay with Java 8 as long as FirstSpirit supports it. This means that the priority of other JDKs is currently low. However, we will look into those warnings and try to fix them for other JDKs.

Thanks for your report

**Comment by grouchal** [comment]
👍 

```
INFO fs-cli version 2.6.2 / git hash 360dcd5282cc7e4ff2af9a9c50cfcb7b0b8cd63b
 INFO Build for FirstSpirit version 5.2.200807
 INFO Using FirstSpirit Access API version 5.2.201008.79347 (fs-access.jar)
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.espirit.moddev.cli.reflection.ReflectionUtils (file:/opt/fs-cli/lib/fsdevtools-cli-2.6.2.jar) to field sun.reflect.annotation.AnnotationInvocationHandler.memberValues
WARNING: Please consider reporting this to the maintainers of com.espirit.moddev.cli.reflection.ReflectionUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by kafmann-e-spirit** [comment]
Hey @m-kotowski ,
I just want to inform you that we fixed this bug with Release 2.6.10 (added support Java 16)

Thanks for your contribution!
